components/wizard/overview.md

```diff
@@ -0,0 +1,153 @@
+                                </Template>
+                            </FormItem>
+                            <FormItem Field="@nameof(User.Password)">
+                                <Template>
+                                    <label for="pass" class="k-label k-form-label">Password*:</label>
+                                    <TelerikTextBox Id="pass" @bind-Value="@UserModel.Password" Password="true"></TelerikTextBox>
+                                    <TelerikValidationMessage For="@(() => UserModel.Password)"></TelerikValidationMessage>
+                                </Template>
+                            </FormItem>
+                            <FormItem Field="@nameof(User.AcceptTerms)" />
+                        </FormItems>
+                        <FormButtons></FormButtons>
+                    </TelerikForm>
+                </Content>
+            </WizardStep>
+            <WizardStep Label="Attachments" Icon="paperclip">
+                <Content>
+                    @* No Upload functionality is included for brevity. If needed, it can be additionally set up. *@
+                    <TelerikUpload AllowedExtensions="@( new List<string>() { ".pdf", ".docx" } )" />
+                </Content>
+            </WizardStep>
+            <WizardStep Label="Confirmation" Icon="check">
+                <Content>
+                    <h2>Registration completed.</h2>
+                </Content>
+            </WizardStep>
+        </WizardSteps>
+    </TelerikWizard>
+}
+
+@code {
+
+    public bool ShowWizard { get; set; } = true;
+
+    public int Value { get; set; }
+
+    public TelerikForm RegisterForm { get; set; }
+    public User UserModel { get; set; } = new User();
+
+    public void OnRegistrationStepChange(WizardStepChangeEventArgs args)
+    {
+        var isFormValid = RegisterForm.IsValid();
+        if (!isFormValid)
+        {
+            args.IsCancelled = true;
+        }
+    }
+
+    public void OnFinishHandler()
+    {
+        ShowWizard = false;
+    }
+
+    public class User
+    {
+        [Required]
+        public string FirstName { get; set; } = "John";
+
+        [Required]
+        public string LastName { get; set; } = "Smith";
+
+        [Required]
+        public string Email { get; set; } = "email@domain.com";
+
+        [MinLength(3, ErrorMessage = "The password should be at least 3 characters.")]
+        [Required]
+        public string Password { get; set; }
+
+        public DateTime? BirthDate { get; set; }
+
+        [Range(typeof(bool), "true", "true", ErrorMessage = "You must agree with the terms.")]
+        [Display(Name = "Accept Terms and Conditions")]
+        public bool AcceptTerms { get; set; }
+    }
+}
+````
+
+## Features
+
+* `StepperPosition` - `WizardStepperPosition` - Specify where the stepper is rendered against the wizard content. The default is `Top`. Read more about its configuration in the [Layout]() article.
```

Comment:
missing slug

Maybe paraphrase the second sentence as See the Layout article for more information.

Reply:
Done